rtpsender_test.go

```diff
@@ -401,3 +401,75 @@ func Test_RTPSender_Add_Encoding(t *testing.T) {
 
 	assert.NoError(t, peerConnection.Close())
 }
+
+// nolint: dupl
+func Test_RTPSender_FEC_Support(t *testing.T) {
+	t.Run("FEC SSRC by Default", func(t *testing.T) {
+		track, err := NewTrackLocalStaticSample(RTPCodecCapability{MimeType: MimeTypeVP8}, "video", "pion")
+		assert.NoError(t, err)
+
+		peerConnection, err := NewPeerConnection(Configuration{})
+		assert.NoError(t, err)
+
+		rtpSender, err := peerConnection.AddTrack(track)
+		assert.NoError(t, err)
+
+		assert.NotZero(t, rtpSender.GetParameters().Encodings[0].FEC.SSRC)
```

Comment:
Missing something here. I did not see where FEC is registered by default. Is the default registration of FEC not part of this PR?

Reply:
you weren't missing anything, my mistake! Test was inverted by accident, for now going to leave FEC disabled by default (but let users enable it)

Just want RTX enabled by default. Will be updating interceptor to send that appropriately.